doc/Makefile.am

```diff
@@ -112,7 +112,7 @@ $(DOCDIR)/userguide/index.html: 	$(DOCDIR) userguide.xml userguide_desc.xml \
 		fi; \
 	fi
 # try xmlto ...
-	test -r $(abs_srcdir)/userguide.xml
+	test -r "$(abs_srcdir)/userguide.xml"
```

Comment:
I suppose the lines below should be updated too

Reply:
Honestly I am not sure because I only guessed at what to change. ;)

For me the build works without further changes. I can only assume this has to do with one line using abs_srcdir and the other line using srcdir?

Addendum: I added some echos to the makefile. During the build srcdir is
`.`
while abs_srcdir is
`/home/jan/freetds 1.3/freetds/doc`.
This is why the other lines don't need extra quotes:

~~~
echo "-------------"
-------------
echo "abs_srcdir: /home/jan/freetds 1.3/freetds/doc"
abs_srcdir: /home/jan/freetds 1.3/freetds/doc
echo "srcdir: ."
srcdir: .
echo "-------------"
-------------
test -r "/home/jan/freetds 1.3/freetds/doc/userguide.xml"
test -r userguide.xml || ln -s ./userguide.xml userguide.xml
test -r userguide.xsl || ln -s ./userguide.xsl userguide.xsl
test -r userguide_desc.xml || ln -s ./userguide_desc.xml userguide_desc.xml
~~~

I am not sure, why abs_srcdir gets used instead of srcdir.